public/app/core/history/richHistoryLocalStorage.ts

```diff
@@ -0,0 +1,172 @@
+import RichHistoryStorage, {
```

Comment:
there are a bunch of return type definitions in the `RichHistoryLocalStorage` which are the same defined in the `RichHistoryStorage` interface. why are we rewriting them here? doesn't ts infer it properly?

also, according to https://github.com/grafana/grafana/blob/eb4c3a4485e99a5dadf41e8d0536a41f51bb4a0a/contribute/style-guides/frontend.md the name of this file (and some of the others in this directory) should be in PascalCase

Reply:
Good point! I've renamed the files and removed inferred return types in cebad27882e468934f0fa92cb303545b24137786